torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -49,8 +70,11 @@ def get_num_channels(image: Union[features.ImageTypeJIT, features.VideoTypeJIT])
 
 
 def get_spatial_size_image_tensor(image: torch.Tensor) -> List[int]:
-    width, height = _FT.get_image_size(image)
-    return [height, width]
+    hw = list(image.shape[-2:])
```

Comment:
Same as above. Is there a reason to replicate the logic?

Reply:
Let's keep the discussion in one place above.